corehq/apps/app_manager/tests/data/bulk_app_translation/download/app.json

```diff
@@ -1 +1,447 @@
+{
```

Comment:
This change is similar to https://github.com/dimagi/commcare-hq/pull/6503/files#diff-d9bc9308147e25e5268d7141d167306dR280
Plus one line to many lines.